pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -698,6 +698,9 @@ type NodeGroup struct {
 	// +optional
 	EBSOptimized *bool `json:"ebsOptimized,omitempty"`
 
+	// T3Unlimited enables [T3 Unlimited](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/burstable-performance-instances-unlimited-mode.html), valid only for T-type instances
+	T3Unlimited *bool `json:"t3Unlimited,omitempty"`
```

Comment:
I would definitely prefer to have it called `CPUCreditsUnlimited` or similar. @martina-if @cPu1

Reply:
Yes, I called it `t3Unlimited` only because it will be easier to google.

Reply:
Or even `cpuCredits: string`

Reply:
Yes, `cpuCredits: string` looks best of all. I'll change this after stabilization if main functionality.